gradle/libs.versions.toml

```diff
@@ -39,11 +39,11 @@ kotlin = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk7", version.ref = "ko
 kotlin-gradle-plugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
 maps-ktx-std = { module = "com.google.maps.android:maps-ktx", version.ref = "maps" }
 maps-ktx-utils = { module = "com.google.maps.android:maps-utils-ktx", version.ref = "maps" }
```

Comment:
We should only have to load maps-utils-ktx and not maps-utils, but we should make this change after https://github.com/googlemaps/android-maps-ktx/issues/206 is fixed to bring maps-utils-ktx up to date.